python/smashbox/utilities/__init__.py

```diff
@@ -9,7 +9,7 @@
 
 ######## TEST SETUP AND PREPARATION
 
-def reset_owncloud_account(reset_procedure=None):
+def reset_owncloud_account(reset_procedure=None,numTestUsers=None):
```

Comment:
It is impossible to specify that there is only one account which  name should not be altered. I propose to change it in the follow way: value None should mean that there is only one account name as specified by oc_account_name. Any other value (number) means that we enter the numbered account mode and that the number will be appended to oc_account_name. In the test code if you want to use the default number of users you should specify that explicitly: (...., numTestUsers=config.oc_number_test_users)

How does it sound for you?